apps/dh/api-dh/source/DataHub.WebApi/Modules/ProcessManager/Calculations/Client/WholesaleClientAdapter.cs

```diff
@@ -132,10 +133,10 @@ private OrchestrationInstanceTypedDto<CalculationInputV1> MapCalculationDtoToOrc
 
     private OrchestrationInstanceLifecycleDto MapCalculationDtoToOrchestrationInstanceLifecycleDto(CalculationDto c) =>
         new(
-            new UserIdentityDto(c.CreatedByUserId, Guid.Empty), // ActorId is not used
+            new UserIdentityDto(c.CreatedByUserId, null!, null!), // Actor number/role is not used
             MapCalculationOrchestrationStateToOrchestrationInstanceLifecycleState(c.OrchestrationState),
             MapCalculationOrchestrationStateToOrchestrationInstanceTerminationState(c.OrchestrationState),
-            c.OrchestrationState == CalculationOrchestrationState.Canceled ? new UserIdentityDto(c.CreatedByUserId, Guid.Empty) : null,
+            c.OrchestrationState == CalculationOrchestrationState.Canceled ? new UserIdentityDto(c.CreatedByUserId, null!, null!) : null, // Actor number/role is not used
```

Comment:
as scared as I am of `null!` statements, this should work - we do not (currently) use the actor information in `UserIdentityDto`. Hopefully we can get rid of the adapter soon.